icdc_schema.py

```diff
@@ -298,8 +300,10 @@ def _get_item_type(self, item_type):
             return {PROP_TYPE: self.map_type(item_type)}
         elif isinstance(item_type, list):
             enum = set()
-            for t in item_type:
-                if not re.search(r'://', t):
+            r_url = re.compile(r"://")
+            url_list = list(filter(r_url.search, item_type))
+            if not len(item_type) != 1 or len(url_list) != 1:
```

Comment:
There are too many "not", "!" and "or", and make my head spin :-( Could you add some parentheses or reverse the logic to make the logic more clear? Thanks!